src/App.js

```diff
@@ -1,28 +1,27 @@
-import React from 'react';
-import { NavBar } from './pages/globalComponents/navBar'
-import { Home } from './pages/homePage';
-import { About } from './pages/aboutUs';
-import { Members } from './pages/members';
-import { Contact } from './pages/contactUs';
-import { Join } from './pages/joinUs';
-import { Login } from './pages/login';
+import './App.css';
+import {BrowserRouter, Routes, Route} from 'react-router-dom'
+import {HomePage, AboutUs, ContactUs, JoinUs, Members, Login, NotFound} from './pages'
+import {NavBar} from './pages/globalComponents'
```

Comment:
@Chaitanya-087 Kindly Import Navbar from './pages' only,
as you can export this NavBar from './pages/index.js'